src/main/kotlin/no/nav/syfo/client/pdl/GeografiskTilknytningNotFoundException.kt

```diff
@@ -0,0 +1,6 @@
+package no.nav.syfo.client.pdl
+
+private const val messageStart =
+    "Request to get Geografisk Tilknytning from PersonDataLosningen was succesful, but no Geografisk Tilknytning was found in response."
+
+class GeografiskTilknytningNotFoundException : RuntimeException(messageStart)
```

Comment:
Denne brukes ikke noe sted?

Reply:
Ja, kanskje vi trenger den i neste omgang, men kan slette den for nå.